### PR TITLE
*: add upgrade FAQs in 2.1

### PR DESCRIPTION
**Pull request by CaitinChen**
Via https://github.com/pingcap/docs-cn/pull/1185
@lilin90 @yikeke @crazycs520 PTAL

### CODE REVIEW

File: TOC.md [hunk 1]
```diff
@@ -109,7 +109,7 @@
     - [Upgrade the Component Version](op-guide/ansible-deployment-rolling-update.md#upgrade-the-component-version)
     - [TiDB 2.0 Upgrade Guide](op-guide/tidb-v2.0-upgrade-guide.md)
     - [TiDB 2.1 Upgrade Guide](op-guide/tidb-v2.1-upgrade-guide.md)
-    - [FAQs After Upgrading](op-guide/upgrade-FAQ.md)
+    - [FAQs After Upgrading](op-guide/upgrade-faq.md)
```
Comment (lilin90) [inline TOC.md]: Is `FAQs After Upgrade` more appropriate? Generally, we expect "after upgrading to ..." or "after upgrading sth.".
```suggestion
    - [FAQs After Upgrade](op-guide/upgrade-faq.md)
```

File: v2.1/README.md [hunk 2]
```diff
@@ -109,6 +109,7 @@
     - [Upgrade the Component Version](op-guide/ansible-deployment-rolling-update.md#upgrade-the-component-version)
     - [TiDB 2.0 Upgrade Guide](op-guide/tidb-v2.0-upgrade-guide.md)
     - [TiDB 2.1 Upgrade Guide](op-guide/tidb-v2.1-upgrade-guide.md)
+    - [FAQs After Upgrading](op-guide/upgrade-faq.md)
```
Comment (lilin90) [inline v2.1/README.md]:
```suggestion
    - [FAQs After Upgrade](op-guide/upgrade-faq.md)
```

File: v2.1/op-guide/upgrade-faq.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
+---
+title: FAQs After Upgrading
```
Comment (lilin90) [inline v2.1/op-guide/upgrade-faq.md]:
```suggestion
title: FAQs After Upgrade
```

File: v2.1/op-guide/upgrade-faq.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+summary: This document lists some FAQs and their solutions after upgrading.
+category: deployment
+---
+
+# FAQs After Upgrading
```
Comment (lilin90) [inline v2.1/op-guide/upgrade-faq.md]:
```suggestion
# FAQs After Upgrade
```

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again.

File: v2.1/op-guide/upgrade-faq.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
+---
+title: FAQs After Upgrade
+summary: This document lists some FAQs and their solutions after upgrading.
```
Comment (lilin90) [inline v2.1/op-guide/upgrade-faq.md]:
```suggestion
summary: Learn about the FAQs after upgrading TiDB.
```

**Pull request merged**